### DISCUSSION TITLE
Add missing shortcuts (sixths for example)

### DISCUSSION THREAD

**Discussion by nununo** [rxhanson/Rectangle#560] [Ideas]
Hello,

I'm using a 27" 4K monitor and have been dividing it in sixths. Unfortunately, while Rectangle supports sixths, it doesn't allow me to assign shortcuts to it.

Would it be possible to add shortcuts to the options that currently don't allow them?

Thanks,
Nuno

**Comment by rxhanson** [comment]
Sounds like you just need to click the triangle with the ellipsis in the preferences window to show the remaining shortcut recorders?

I’m planning a UI update that will change the shortcut display and should be more intuitive.

**Reply by nununo** [reply]
Oh my! I never saw that there before. Thank you and sorry. Shortcuts defined. I'm happy again!
